build/quarks-secret/quarks-values.yaml

```diff
@@ -0,0 +1,9 @@
+fullnameOverride: cf-quarks-secret
+
```

Comment:
btw, you might consider using `applyCRD: false`, because in [(v1.0.692) upstream in the `crds.yaml`, it looks like the crds only get templated out if you have `applyCRD: false`](https://github.com/cloudfoundry-incubator/quarks-secret/blob/v1.0.692/deploy/helm/quarks-secret/templates/crds.yaml#L1), because if it's `true`, it looks like (though I haven't tested) the operator itself will push out the crd, though I would think it'd be more desirable to have it be rendered out as plain yaml for diff'ing and transparency purposes (see previous discussion: https://github.com/cloudfoundry/cf-for-k8s/pull/280#discussion_r454697795)

(apologies, not trying to clutter anything, just something I figured I'd say)